tools/run_tests/run_tests.py

```diff
@@ -57,8 +57,8 @@ def __init__(self, config, environ={}):
     self.allow_hashing = (config != 'gcov')
     self.environ = environ
 
-  def job_spec(self, binary, hash_targets):
-    return jobset.JobSpec(cmdline=[binary],
+  def job_spec(self, cmdline, hash_targets):
```

Comment:
run_tests probably needs a documentation scrub in general. If this one method is a blocker for you, I'll add something. For the script as a whole though, that's not going to happen today.